plugins.js

```diff
@@ -241,6 +241,17 @@ var plugins = [
       version: '0.18.0'
     }]
   },
+  {
+    name: 'Blotter',
+    description: 'Writes all buy/sell trades to a blotter CSV file',
+    slug: 'blotter',
+    async: false,
+    modes: ['realtime'],
+    dependencies: [{
+      module: 'fs',
```

Comment:
fs is a build in module, you don't need to specify it as a dependency here.

Reply:
You're right. Dependency removed